src/Rules/Resources/Descriptions.resx

```diff
@@ -147,51 +147,51 @@
   </data>
   <data name="IsContentElementFalseOptional" xml:space="preserve">
     <value>The recommended value of the IsContentElement property for the given control type is false. Please consider if this is an element that should be reported to an assistive technology user as content.</value>
-    <comment>The description of an accessibility problem with an element currently under inspection. {Locked="IsContentElement"}</comment>
+    <comment>The description of an accessibility problem with an element currently under inspection. {Locked="false", "IsContentElement"}</comment>
```

Comment:
@csigs, what's the policy on localizing boolean values? I know that `false.ToString()` in French returns `faux` instead of `false`. What's the guidance for boolean values in docs?